libs/content/workspace/src/tab/svg_editor/selection.rs

```diff
@@ -90,6 +90,7 @@ impl Selection {
                     } else {
                         self.selected_elements = vec![new_selected_el]
                     }
+                    self.current_op = SelectionOperation::Drag;
```

Comment:
this is unrelated to this PR, but i noticed a small bug for the selection module when doing QA.
the bug is that when you press on the edge of an un-selected path you go directly into resize instead of grab mode.